main/config.py

```diff
@@ -13,6 +13,10 @@
 class Config:
     BASE_DIR = typing.cast("Path", settings.BASE_DIR)
 
+    HOT_TASKING_MANAGER_PROJECT_API_URL = (
+        "https://tasking-manager-production-api.hotosm.org/api/v2/projects/{project_id}/queries/aoi/?as_file=false"
+    )
```

Comment:
Maybe we should only define `https://tasking-manager-production-api.hotosm.org/api/v2` here?
@thenav56